src/main/scala/beam/sim/BeamSim.scala

```diff
@@ -291,17 +288,33 @@ class BeamSim @Inject()(
     )
 
     //rename output files generated by matsim to follow the standard naming convention of camel case
-    renameGeneratedOutputFiles(event)
+    val outputFiles = renameGeneratedOutputFiles(event)
+
+    val scenario = event.getServices.getScenario
+    val controllerIO = event.getServices.getControlerIO
+
+    writeScenario(scenario, controllerIO)
+
     outputFilesToDelete.foreach(deleteOutputFile)
 
     def deleteOutputFile(fileName: String) = {
       logger.debug(s"deleting output file: $fileName")
-      Files.deleteIfExists(Paths.get(event.getServices.getControlerIO.getOutputFilename(fileName)))
+      Files.deleteIfExists(Paths.get(controllerIO.getOutputFilename(fileName)))
     }
     BeamConfigChangesObservable.clear()
 
   }
 
+  private def writeScenario(scenario: Scenario, controlerIO: OutputDirectoryHierarchy) = {
+    PopulationCsvWriter.toCsv(scenario, controlerIO.getOutputFilename("population.csv"))
+    VehiclesCsvWriter(beamServices).toCsv(scenario, controlerIO.getOutputFilename("vehicles.csv"))
+    HouseholdsCsvWriter.toCsv(scenario, controlerIO.getOutputFilename("households.csv"))
+    PlansCsvWriter.toCsv(scenario, controlerIO.getOutputFilename("plans.csv"))
+    NetworkNodeCsvWriter.toCsv(scenario, controlerIO.getOutputFilename("networkNode.csv"))
```

Comment:
we only need the network (rename networkMerge.csv to network.csv). networkNode.csv and networkLink.csv are not needed

Reply:
updated